content/community/articles.md

```diff
@@ -2,7 +2,7 @@
 title: "Articles"
 ---
 
-### Camel Videos
+## Camel Videos
```

Comment:
These changes could be on another PR, they don't seem to be related to this.

Reply:
Well, a few of the contents within the Hugo pages weren't perfectly aligned due to which the toc content wasn't being generated accurately. Hence, I had to make changes to these particular files.